### DISCUSSION TITLE
什么叫代公众号调用接口？

### DISCUSSION THREAD

**Discussion by tian-yan** [wechatpy/wechatpy#668] [Q&A] [answered]
据我所知，微信公众号开发接口，分两种类型：
1. 主动调用接口：第三方服务器通过 **`Access_Token`** 向微信服务器发起调用请求，如创建菜单。（需要在微信服务器上获取AppID和AppSecret）
2. 被动调用接口：微信服务器将微信用户的请求转发至第三方服务器，第三方服务器根据业务逻辑进行响应。（需要在微信服务器上配置第三方URL，Token，加密方式等）

那[官方文档](https://wechatpy.readthedocs.io/zh_CN/master/component.html)上提到的 **`代公众号调用接口`** 怎么理解？

**Comment by messense** [comment, marked as answer]
https://developers.weixin.qq.com/doc/oplatform/Third-party_Platforms/2.0/product/Third_party_platform_appid.html

**Reply by tian-yan** [reply]
多谢，原来是腾讯自己引入的概念。
